cacheops/query.py

```diff
@@ -86,11 +88,23 @@ def wrapper(*args, **kwargs):
             cache_key = 'as:' + _get_key(func, args, kwargs, key_extra)
 
             cache_data = redis_client.get(cache_key)
+
```

Comment:
Please, don't add blank lines where they weren't before. This pollutes history and also departs from current terse style.

Also, please remove them with rebase to leave no history of them added and removed.